### PR TITLE
💄(styling): Added global color palette

### PR DESCRIPTION
**Pull request by Ruben-Brits**
Implemented a global colour pallet for use throughout the UI. Also did a small UI change to ensure it works as inteded

### CODE REVIEW

File: apps/client/src/styles.scss [hunk 1]
```diff
@@ -39,5 +39,28 @@ $client-theme: mat.define-light-theme((
 
 /* You can add global styles to this file, and also import other style files */
 
+//Primary Colors
+$primary-1 : #174372;
```
Comment (arneschreuder) [inline apps/client/src/styles.scss:43]: This file should be put into an assets folder, and probably be made into a shared lib?

**Review by arneschreuder — CHANGES_REQUESTED** [review]
Move assets please

**Comment by arneschreuder** [comment]
Build is failing. Please fix issues in ci/cd and then we can get this merged in after you moved the assets

File: apps/client/src/styles.scss (hunk 1 above)
Comment (Ruben-Brits) [reply apps/client/src/styles.scss]: moved into a shared asset lib

**Review by dylan2894 — APPROVED** [review]
Looks good 👍

**Comment by arneschreuder** [comment]
Please add meta (labels, project, milestone etc) to the PR. Then merge.

**Pull request merged**